test/remark.test.ts

```diff
@@ -55,6 +55,12 @@ ruleTester.run('remark', remark, {
       parserOptions,
       filename: path.resolve(__dirname, 'fixtures/async/test.mdx'),
     },
+    {
+      code: '_emphasis_ and __strong__',
+      parser,
+      parserOptions: { ...parserOptions, ignoreRemarkConfig: true },
+      filename: path.resolve(__dirname, 'fixtures/style/test.mdx'),
+    },
```

Comment:
There's a similar test below: https://github.com/mdx-js/eslint-mdx/blob/b1db7640b69538d2f220c0707a866a17b955dea8/test/remark.test.ts#L88

Here we test that the ignoreRemarkConfig just works don't throwing any error.
If the config wouldn't be ignored, this test case wouldn't be invalid.